RoddyCore/src/de/dkfz/roddy/execution/io/ExecutionService.groovy

```diff
@@ -246,44 +246,48 @@ abstract class ExecutionService implements BEExecutionService {
         }
     }
 
+    /** Create somehow valid submission IDs.
```

Comment:
Create low-level issue, this should somehow be in JobManager / BatchEuphoria

Reply:
See https://github.com/TheRoddyWMS/BatchEuphoria/issues/118